src/audio/dai-zephyr.c

```diff
@@ -1619,13 +1626,6 @@ int dai_zephyr_copy(struct dai_data *dd, struct comp_dev *dev, pcm_converter_fun
 		comp_warn(dev, "dai_zephyr_copy(): Copy_bytes %d + free bytes %d < period bytes %d, possible glitch",
 			  copy_bytes, free_bytes, dd->period_bytes);
 
```

Comment:
Hmm, we do skip the glitch detection as well. This has no functional impact, but will affect debugging as the buffer avail/free state is not printed out when we do run out.